### DISCUSSION TITLE
Buttons & Scripts [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5649] [Discord Threads]
> 🙋‍♂️ **SH4RK21**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### @SH4RK21 Currently not supported. In our roadmap  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093904567424516148)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
